static-site/src/components/splash/userGroups.jsx

```diff
@@ -9,7 +9,7 @@ const UserGroups = (props) => {
 
   const colors = [...theme.titleColors];
 
-  const groupCards = props.user.groups.map((group) => {
+  const groupCards = props.user.visibleSources.map((group) => {
```

Comment:
There's a conflict of terms here: the array is one of sources, but the map function calls them groups.

It may be that my off-the-cuff suggestion of `visibleSources` was a poor one and `visibleGroups` would be better.